etc/cp-ksql-server/_base_observations_stream.sql

```diff
@@ -0,0 +1,20 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Register the 'ksql_observations' topic (is created when not exists).
+CREATE STREAM observations (
+    PROJECT VARCHAR KEY,    -- 'KEY' means that this field is part of the kafka message key
+    SUBJECT VARCHAR KEY,
+    SOURCE VARCHAR KEY,
+    `TOPIC` VARCHAR,
+    CATEGORY VARCHAR,
+    VARIABLE VARCHAR,
+    DATE TIMESTAMP,
```

Comment:
I think that there is a `time` column created by jdbc-connector.  Renaming `DATE` to `time` would cause a clash here. What would work is for instance `time_of_observation` or `observation_time`. What do you think?

Reply:
We use TIME in our queries and works with jdbc-connector. But yes `observation_time` works too

Reply:
I renamed `DATE` to `OBSERVATION_TIME` and `END_DATE` to `OBSERVATION_TIME_END`.